android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java

```diff
@@ -377,4 +378,29 @@ public void update(ChatState chatState) {
             }
         });
     }
+
+	@ReactMethod
+	public void areAgentsOnline(Promise promise) {
```

Comment:
No import for Promise, Resulting in Crash, Please fix it, you cant use Promise

Reply:
This is fixed in v0.4.1-beta.7 and v0.4.1-beta.8. What version are you using?

Reply:
Oh! Maybe it’s not. I’ll take a look tomorrow. We had some build issues due to jitpack.io being offline yesterday.

Reply:
yes its not and after npm install 0.4.1-beta.8 is getting installed and causing build error, you need to fix this, in a new PR

Reply:
Active fixes for 0.4.1 can be found in #85 -- `0.4.1-beta.9` is published and now building in our CI. If further fixes are needed, I'll publish more betas until it seems ready.

Reply:
Beta.9 passed compilation, so I'm shipping 0.4.1! -- If you find bugs, please report them.

Reply:
Nice work @fbartho !!